deps/ntlmclient/util.h

```diff
@@ -8,7 +8,8 @@
 
 #ifndef PRIVATE_UTIL_H__
 #define PRIVATE_UTIL_H__
-
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
These changes don't have anything to do with cmake. Can you remove them from this PR, and move them to the upstream project at https://github.com/ethomson/ntlmclient with an explanation of what they do and why they are necessary?

Reply:
I have sent a PR however It was necessary to make the change otherwise the compilation failed